easybuild/easyconfigs/j/Java/Java-8.eb

```diff
@@ -0,0 +1,23 @@
+# Authors:: Jack Perdue <j-perdue@tamu.edu> - TAMU HPRC - http://hprc.tamu.edu
+
+easyblock = 'Bundle'
+
+name = 'Java'
+version = '8'
```

Comment:
This makes quite a lot of changes all in one go - I feel this should be made more prominent somehow.

Reply:
Can you clarify? All we're changing here is using a wrapper for `Java`...

This change in policy will be clearly mentioned in the EasyBuild release notes, and maybe we should also highlight it with a separate post on the mailing list...

Reply:
I don't disagree, just noting that we're making lots of simultaneous changes, so we should communicate this as widely as possible.